core/src/main/java/com/amplifyframework/predictions/models/LandmarkType.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.models;
+
+/**
+ * Types of detectable landmark.
+ */
+@SuppressWarnings("JavadocVariable")
+public enum LandmarkType {    
```

Comment:
The list is specific to facial features, it isn't generic like the name of the enum. How about we just call this `FacialFeature`. `FacialFeature.LEFT_EYE`, etc.

Reply:
I agree. I was thinking about this too, but was conservative about wildly renaming feature names from iOS